_posts/2020-06-27-lsp-explained.md

```diff
@@ -0,0 +1,183 @@
+---
+title: The Liskov Substitution Principle Explained
+categories: [craft]
+date: 2020-06-27 05:00:00 +1100
+modified: 2020-06-27 05:00:00 +1100
+author: saajan
+excerpt: "LSP is a very useful idea both when developing new applications and modifying existing ones. This article explains what it is, why it's important and how to use it."
+image:
+  auto: 0071-disk
+---
+
+This article gives a quick intro to the Liskov Substitution Principle (LSP), why it's important, and how we can use it to validate our object-oriented designs. We'll also see some examples and learn how to correctly identify violations of LSP.
+
+{% include github-project.html url="https://github.com/thombergs/code-examples/tree/master/solid" %}
+
+# What is LSP?
```

Comment:
First section level starts with "##", please add one "#" to each section.

Reply:
Fixed.